packages/server-wallet/src/wallet/index.ts

```diff
@@ -634,7 +646,10 @@ export class Wallet extends EventEmitter<WalletEvent>
 
   // ChainEventSubscriberInterface implementation
   onHoldingUpdated(arg: HoldingUpdatedArg): void {
```

Comment:
I realise that this change isn't part of this PR, but I'm wondering if this is the best name for this method. As far as I'm aware `onXXX` methods are normally exposed on the event emitter:
```ts
emitter.on('some-event', (event) => doAction(event))
// or
emitter.onSomeEvent((event) => doAction(event))
```
The `onXXX` method exists on the emitter, and is called by the subscriber, who passes in a callback that is used to bind an action to that event.

In this case it's the other way around. The emitter is calling the `onHoldingUpdated` method on the subscriber with the event data. Of course this works, it just seems to run contrary to the normal pattern around events and `onXXX` methods, which could be confusing. For example, it took me a while to figure out what was going on here, as the naming pushed me in a different direction.

Maybe something like `holdingUpdated` or `holdingChanged` would work?

Reply:
Good point! Let's change the name.

Reply:
Addressed in 5bdbb0c3630656341274fa0c7323a24d25721a84